templates/taxi/car_detail.html

```diff
@@ -2,6 +2,8 @@
 
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <a href="{% url "taxi:car-update" pk=car.id %}">Update</a>
+  <a href="{% url "taxi:car-delete" pk=car.id %}">Delete</a>
```

Comment:
Ensure that the URL pattern for 'car-delete' includes the primary key (pk) parameter as expected in the `urls.py` file. Verify that the URL pattern is defined as 'cars/delete/<int:pk>/' to match this usage.